easybuild/easyconfigs/m/Mmg/Mmg-5.8.0-gompi-2024a.eb

```diff
@@ -0,0 +1,52 @@
+easyblock = 'CMakeMake'
+
+name = 'Mmg'
+version = '5.8.0'
+
+homepage = 'https://www.mmgtools.org/'
+description = """
+Mmg is an open source software for simplicial remeshing. It provides 3 applications and 4 libraries:
+the mmg2d application and the libmmg2d library: adaptation and optimization of a two-dimensional
+triangulation and generation of a triangulation from a set of points or from given boundary edges
+the mmgs application and the libmmgs library: adaptation and optimization of a surface triangulation
+and isovalue discretization the mmg3d application and the libmmg3d library: adaptation and optimization
+of a tetrahedral mesh and implicit domain meshing the libmmg library gathering the libmmg2d,
+libmmgs and libmmg3d libraries."""
+
+toolchain = {'name': 'gompi', 'version': '2024a'}
+
+source_urls = ['https://github.com/MmgTools/mmg/archive']
+sources = ['v%(version)s.tar.gz']
+patches = ['%(name)s-5.7.2_fix-test-fail.patch']
+checksums = [
+    {'v5.8.0.tar.gz': '686eaab84de79c072f3aedf26cd11ced44c84b435d51ce34e016ad203172922f'},
+    {'Mmg-5.7.2_fix-test-fail.patch': 'c01cbfadad8c70a8eb3766cc73ab09edd331dda1f445e14c547f28d839c973e6'},
+]
+
+builddependencies = [
+    ('CMake', '3.29.3'),
+]
+
+dependencies = [('SCOTCH', '7.0.6')]
+
+# common configopts for static, shared library builds
+_base_configopts = ' '.join([
```

Comment:
```
-- Found Perl: /usr/bin/perl (found version "5.26.3")
```

Perl is used to generate the Fortran headers.

Reply:
done in https://github.com/easybuilders/easybuild-easyconfigs/pull/22999/commits/025ae5aed28698d66aede11b385b567f5e51c03f